src/config/index.ts

```diff
@@ -14,7 +14,8 @@ export const NODE_ENV = getEnvironmentValue("NODE_ENV");
 export const CDN_HOST = getEnvironmentValue("CDN_HOST");
 
 // ROUTING PATH
-export const LANDING_URL = "/";
+export const REGISTER_AN_OVERSEAS_ENTITY_URL = "/register-an-overseas-entity";
+export const LANDING_URL = REGISTER_AN_OVERSEAS_ENTITY_URL;
```

Comment:
Just wondering, why have LANDING_URL and why not just use/reference the new REGISTER_AN_OVERSEAS_ENTITY_URL?

Reply:
Hi, just naming convention, I could have used the REGISTER_AN_OVERSEAS_ENTITY_URL but it is going to be the first section of all our endpoints, so I included it in the landing page as well. It will have been overkilling having an endpoints like this with just  "/" at the end 
`export const LANDING_URL = REGISTER_AN_OVERSEAS_ENTITY_URL + "/";`

Reply:
So, yes, I like the REGISTER_AN_OVERSEAS_ENTITY_URL name but still not sure why we bother keeping a LANDING_URL constant which is exactly the same as REGISTER_AN_OVERSEAS_ENTITY_URL. Maybe I'm missing something? happy to have a quick call...

Reply:
Lets have a chat :)